bitnami/contour/Chart.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: contour
+description: Contour Ingress controller for Kubernetes
+appVersion: 1.2.1
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+keywords:
+  - ingress
+  - envoy
+  - contour
+home: https://projectcontour.io
+icon: https://d33wubrfki0l68.cloudfront.net/b92d8706535e6f2b71f90645fe9ab7b418fb8ca8/ea52e/img/contour.svg
+sources:
+- https://github.com/projectcontour/contour
+- https://github.com/envoyproxy/envoy
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.3
+maintainers:
+  - name: cellebyte
+    url: https://github.com/Cellebyte
```

```suggestion
apiVersion: v1
name: contour
description: Contour Ingress controller for Kubernetes
appVersion: 1.2.1
keywords:
  - ingress
  - envoy
  - contour
home: https://projectcontour.io
icon: https://d33wubrfki0l68.cloudfront.net/b92d8706535e6f2b71f90645fe9ab7b418fb8ca8/ea52e/img/contour.svg
sources:
  - https://github.com/projectcontour/contour
  - https://github.com/envoyproxy/envoy
  - https://github.com/bitnami/bitnami-docker-contour
version: 0.1.3
maintainers:
  - name: cellebyte
    url: https://github.com/Cellebyte
  - name: Bitnami
    email: containers@bitnami.com
```

What do you think about the above changes? Mainly add maintainer, source, remove comments and `apiVersion: v1` to be consistent with the other charts in the catalog